bitnami/wordpress/templates/configmap.yaml

```diff
@@ -0,0 +1,8 @@
+{{- if .Values.customPostInitScripts }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "wordpress.fullname" . }}-custominit-configmap
```

Comment:
Some times the length of the name causes issues, I think this name will be too long due to the length fixed part `-custominit-configmap`. Could you check that and in that case, use a short name, please?

Reply:
Fair point; changed name to {{ template "wordpress.fullname" . }}-postinit